packages/homepage/src/screens/home/hero/Content.js

```diff
@@ -64,7 +86,11 @@ const Content = () => (
         }}
         href="/s"
       >
-        Create Sandbox, it’s free
+        {freeWordingA ? (
+          'Create Sandbox, it’s free'
```

Comment:
@danilowoz I think here something is wrong. Please if you have some time could you take a look?